test/tests/redfish10/test_redfish10_system_reset_base.py

```diff
@@ -152,8 +145,33 @@ def workflow_tasklist_status_poller(tasklist, tasktype, timeout=180):
 
     return task_errorlist
 
+
+def get_rackhd_nodetype(nodeid):
```

Comment:
@anhou This PR is just for pulling the 1.1 APIs out of these two specific scripts that are redfish tests and included in FIT smoke. We are adding more PRs to address non-smoke FIT scripts and another story exists to convert the test_api_utils.py script - RAC-4452.  :-)  
Also, if you dig into the script, the three rackhd20 scripts listed above use api/2.0/nodes within get_rackhd_nodetype.  This function is in test_api_utils, and once updated to 2.0, we should fix the other scripts to use it I think.